tests/end2end-tests/graphstorm-ec/mgpu_test.sh

```diff
@@ -119,6 +119,8 @@ python3 check_infer.py --train_embout /data/gsgnn_ec/emb/ --infer_embout /data/g
 
 error_and_exit $?
 
+rm -fr /data/gsgnn_ec/*
```

Comment:
do you try to remove the files after every test?

Reply:
You are right. Those files will not be used after the test.